pkg/commands/deps.go

```diff
@@ -112,10 +115,24 @@ If the image was not built using ko, or if it was built without embedding depend
 				cmd := exec.CommandContext(ctx, "go", "version", "-m", n)
 				cmd.Stdout = os.Stdout
 				cmd.Stderr = os.Stderr
+				if spdx {
```

Comment:
Consider normalizing the vanilla output of `ko deps` so instead of `/tmp/garbage/` is prints `/ko-app/blah`?

Reply:
Done.